pkg/spec/config_linux.go

```diff
@@ -46,18 +46,23 @@ func devicesFromPath(g *generate.Generator, devicePath string) error {
 		return errors.Wrapf(err, "cannot stat %s", devicePath)
 	}
 	if st.IsDir() {
-		if len(devs) > 2 {
-			return errors.Wrapf(unix.EINVAL, "not allowed to specify destination with a directory %s", devicePath)
-		}
 		found := false
+		destSpecified := len(devs) > 1 && devs[1][0] == '/'
```

Comment:
Could this crash if len(devs[1]) == 0? I don't know how we generate the devs array

Reply:
Oh, that's a good catch. It can crash, see a minimal example: https://play.golang.org/p/Yk52BfKvh2q